Update Material 3 progress indicators and slider guides for theme migration (#11733)
flutter/website: src/content/release/breaking-changes/updated-material-3-progress-indicators.md

),
```

> To opt into the updated design spec for the `LinearProgressIndicator`, for the
> entire app, set the `ProgressIndicatorThemeData.year2023` flag to `false` in
> the `MaterialApp`:

Reviewer: Consider adjusting the wording so "entire app" is earlier on in the sentence. Perhaps something like:

```suggestion
To update your entire app to use the updated `LinearProgressIndicator` design,
set the `ProgressIndicatorThemeData.year2023` property to `false` in your `MaterialApp`.
```

```dart highlightLines=2
return MaterialApp(

>   theme: ThemeData(progressIndicatorTheme: const ProgressIndicatorThemeData(year2023: false)),

Reviewer: I tried this out and it's saying the `year2023` parameter and corresponding property are deprecated?

Author: Yes, we added this flag with deprecation a few months ago.

Reviewer: I see. It's a bit confusing though, as it's a deprecation warning, but in this case, you don't want to remove it.

Should we add an ignore comment to these snippets with an explanation of why the diagnostic is ignored?

Author: I didn't mean they should be ignoring the message but rather being aware of this migration steps. This flag is deprecated since we don't want people to think this will be a permanent flag. Adding message to ignore will defeat the purpose of deprecation.

>         /// ...
>         LinearProgressIndicator(
>           year2023: false,
>           value: 0.5,
>         ),
>         /// ...

Reviewer: Use inline comments here as `///` is for doc comments.

```suggestion
        // ...
        LinearProgressIndicator(
          year2023: false,
          value: 0.5,
        ),
        // ...
```
